pandas/plotting/_core.py

```diff
@@ -1614,6 +1614,11 @@ def scatter(self, x, y, s=None, c=None, **kwargs):
 
               .. versionchanged:: 1.1.0
 
+        size : str, int or array-like, optional
+                The color of each point. Alias for `s`.
+                `s` and `size` aren't allowed at the same time.
+                .. versionadded:: 1.4.1
+
```

Comment:
@attack68 thoughts on how this should be documented?

In the original issue ( https://github.com/pandas-dev/pandas/issues/44670#issuecomment-981663172 ) I'd suggested to just silently accept `size` and `color` as aliases without documenting them, not sure they need an extra section in the docs.

Don't know what's best though, I think aliases are quite rare in pandas

Reply:
I would maybe just add a one-liner to the already documented `s` arg:

"`size` is also accepted as an alternate keyword argument for `s` for consistency with other methods."

Reply:
yeah sounds good

Reply:
I will add this to my changes.